File: app/src/main/java/com/mailerdaemon/app/events/PostModel.java
package com.mailerdaemon.app.events;

import com.google.firebase.firestore.ServerTimestamp;

import java.util.Date;

import lombok.Data;

@Data
public class PostModel {
  private String heading;
  private String details;
  private String photo;
  @ServerTimestamp
  private Date date;

}

File: app/src/main/java/com/mailerdaemon/app/events/PostModel.kt
package com.mailerdaemon.app.events

import com.google.firebase.firestore.ServerTimestamp
import java.util.Date

class PostModel(
    var heading: String? = null,
    var details: String? = null,
    var photo: String? = null,
    @ServerTimestamp
    var date: Date? = null
)
